src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5011,4 +5008,19 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeForwardedToMultipleAssemblies" xml:space="preserve">
     <value>Module '{0}' in assembly '{1}' is forwarding the type '{2}' to multiple assemblies: '{3}' and '{4}'.</value>
   </data>
+  <data name="ERR_BadDocumentationMode" xml:space="preserve">
+    <value>Provided Documentation Mode is unsupported or invalid: '{0}'.</value>
+  </data>
+  <data name="WRN_BadSourceCodeKind" xml:space="preserve">
+    <value>Provided Source Code Kind is unsupported or invalid: '{0}'</value>
+  </data>
+  <data name="ERR_BadLanguageVersion" xml:space="preserve">
+    <value>Provided Language Version is unsupported or invalid: '{0}'.</value>
+  </data>
+  <data name="WRN_BadSourceCodeKind_Title" xml:space="preserve">
+    <value>Provided Source Code Kind is unsupported or invalid</value>
+  </data>
+  <data name="WRN_DefineIdentifierRequired_Title" xml:space="preserve">
```

Comment:
>WRN_DefineIdentifierRequired_Title [](start = 14, length = 34)

Consider keeping this string next to WRN_DefineIdentifierRequired, where it was before.